content/tutorials/managing-config-and-secrets/index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: "Managing Configuration and Secrets"
+title_tag: "Managing Configuration and Secrets"
+layout: single
+
+# A succinct description of the tutorial. It appears on the Tutorials home and collection pages.
+description: Here is a brief description of what this tutorial's all about.
+
+# A similar description used for search results and social-media previews.
+meta_desc: Here is a brief description of what this tutorial's all about.
+
+# An image for the tutorial. It appears on tutorial page and in social-media previews.
+meta_image: config-secrets-meta.png
+
+# The order in which the tutorial appears in most lists. Order is ascending, so higher numbers
+# mean the tutorial will appear further down the list. Positive integers only.
+weight: 999
+
+# A brief summary of the tutorial. It appears at the top of the tutorial page. Markdown is fine.
+summary: |
+    In Pulumi, an environment is a collection of values intended to capture the configuration values needed to work with a particular environment. These can be raw values like server names, environment types, region names and so on. They can also be sensitive values such as database passwords or service tokens.
+    
+    In this tutorial, we'll demonstrate how to create and utilize configuration and secret values in Pulumi.
+
+# A list of three to five things the reader will have learned by the end of the tutorial.
+youll_learn:
+    - How to create a raw configuration value
+    - How to create a secret configuration value
+    - How to view configuration details in the CLI
+    - How to access configuration values from within Pulumi program code
+
+# A list of tutorial prerequisites. Markdown is fine. Keep it simple; no need to be exhaustive here.
+prereqs:
+    - The [Pulumi CLI](/docs/install/)
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - One of Pulumi’s [supported language runtimes](/docs/languages-sdks/) installed
+
+# The estimated time, in minutes, for new users to complete the topic.
+estimated_time: 10
+
+# # An optional list of collections this tutorial should be belong to. Collections are defined in data/tutorials/collections.yaml.
+# collections:
+#     - some-non-existent-collection
+---
+
+## Create a new project
```

Comment:
In the "Inspecting Infrastructure" tutorial, we refer to this step as optional (e.g. either use an existing project or create a new one). That said, tagging this as optional in that tutorial, especially as the first step, felt a little awkward. I think we should be consistent across the tutorials and introduce this step as "Next, either create a new project or navigate to the code for an existing one. To create a project...".

Reply:
I almost wonder if we should just keep it simple rather than adding the indication that it can be run against existing code.